5_databases/week1/homework.sql

```diff
@@ -0,0 +1,52 @@
+USE hyf_lesson1;
+
+-- 1 Find out how many tasks are in the task table
+
+SELECT COUNT (title) FROM task 
+
+-- 2 Find out how many tasks in the task table do not have a valid due date
+
+SELECT * FROM task WHERE due_date IS NULL 
+
+-- 3 Find all the tasks that are marked as done
+
+SELECT * FROM task WHERE status_id = 3 
+
+-- 4 Find all the tasks that are not marked as done
+
+SELECT * FROM task WHERE status_id != 3 
+
+-- 5 Get all the tasks, sorted with the most recently created first
+
+SELECT * FROM task ORDER BY created 
+
+-- 6 Get the single most recently created task
+
+SELECT * FROM task ORDER BY created LIMIT 1 
+
+-- 7 Get the title and due date of all tasks where the title or description contains database
+
+SELECT title, due_date FROM task WHERE title LIKE "%database%" 
+
+-- 8 Get the title and status (as text) of all tasks
+
+SELECT task.title, status.name FROM task JOIN status 
+
+-- 9 Get the name of each status, along with a count of how many tasks have that status
+
+SELECT
```

Comment:
This is the same problem as above, it returns the tasks 3 times. The current output is:


```
+-------------+----------------+
| name        | COUNT(task.id) |
+-------------+----------------+
| Done        |             35 |
| In progress |             35 |
| Not started |             35 |
+-------------+----------------+
```

If you fix the SQL statement above this should be fixed as well.